collector/lib/KernelDriver.cpp

```diff
@@ -0,0 +1,138 @@
+#include "KernelDriver.h"
+
+#include "FileSystem.h"
+#include "SysdigService.h"
+#include "Utility.h"
+
+extern "C" {
+#include <sys/stat.h>
+}
+
+#include <fstream>
+#include <iterator>
+
+#define init_module(module_image, len, param_values) syscall(__NR_init_module, module_image, len, param_values)
+#define delete_module(name, flags) syscall(__NR_delete_module, name, flags)
+
+namespace collector {
+
+namespace {
+
+std::string constructArgsStr(std::unordered_map<std::string, std::string>& args) {
```

Comment:
[nit] It seems that the style within collector is to start each word in a function with a capital letter including the first.

Reply:
I believe that's only true for public functions/methods, and everything else starts with a lowercase letter, similar to Go